src/theme/components/alphabetFilter.ts

```diff
@@ -3,13 +3,17 @@ import { defaultElementSizes } from "./global";
 const AlphabetFilter = {
   baseStyle: {
     button: {
+      color: "ui.link.primary",
       minWidth: { base: defaultElementSizes.mobileFieldHeight, md: "auto" },
       height: { base: defaultElementSizes.mobileFieldHeight, md: "auto" },
       padding: "1px 4px",
       margin: "2px 1px",
       fontWeight: "bold",
       fontSize: "heading.callout",
       lineHeight: "1",
+      _hover: {
```

Comment:
Looks like we also need too remove the pointer cursor for the disabled state.  Add this to `_disabled`:
```
cursor: "not-allowed”
```